_posts/en/newsletters/2020-07-08-newsletter.md

```diff
@@ -0,0 +1,227 @@
+intends to change.
+
+{% include functions/details-list.md
+  q0="What is the importance of `addr` relay and specifically the `getaddr`/`addr` protocol?"
+  a0="`addr` relay is used for nodes to find new potential peers on the P2P network."
+  a0link="https://bitcoincore.reviews/18991.html#l-34"
+
+  q1="Which properties of `addr` relay are important?"
+  a1="Nodes need to learn about a diverse set of peers with good uptime that were online recently."
+  a1link="https://bitcoincore.reviews/18991.html#l-57"
+
+  q2="How can a spy use `addr` messages to infer network topology?"
+  a2="There are potentially multiple ways to infer topology from `addr` messages, but the most discussed method was scraping nodes' address managers (addrman) to determine how an address record is spread across the network, and whether any nodes have a unique timestamp for that address record (indicating that they're probably directly connected to that address). This is the method that was used in the [Coinscope paper][]."
+  a2link="https://bitcoincore.reviews/18991.html#l-129"
+
+  q3="What could a malicious actor do if they were able to map the entire P2P topology?"
+  a3="Knowing the entire P2P network topology makes it easier to carry out attacks such as network partitions or [eclipse attacks][topic eclipse attacks]."
+  a3link="https://bitcoincore.reviews/18991.html#l-176"
+
+  q4="Is it a problem if nodes cache responses to `getaddr` messages, and serve records that are old?"
+  a4="Probably not. There isn't much churn on the P2P network, so old records are usually still valid."
+  a4link="https://bitcoincore.reviews/18991.html#l-91"
+
+  q5="Does this PR prevent the unique-timestamp topology inference?"
+  a5="No. This PR makes it more difficult to scrape a node's address manager (addrman), but does not change the timestamps on relayed address records. A future PR could make additional changes to address the unique-timestamp inference."
+  a5link="https://bitcoincore.reviews/18991.html#l-263"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Eclair 0.4.1][] this new release adds support for
+  `option_static_remotekey` (though disabled by default), which can
+  simplify backups (see [Newsletter #67][news67 bolts642]).  It also
+  enables sending [multipath payments][topic multipath payments] (MPP)
+  by default, uses a new MPP splitting algorithm, offers beta support
+  for using the PostgreSQL database, and better manages feerate
+  mismatches between your node and your peers---all changes described in
+  more detail below in parts of the *Notable Changes* section of this
+  newsletter.
+
+- [LND 0.10.2-beta.rc4][lnd 0.10.2-beta]: this release candidate for an
+  LND maintenance release is now available for testing.  It includes
```

Comment:
perhaps replace 

"release candidate for an LND maintenance release"

with

"LND maintenance release candidate"